build/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG ARCH
+FROM golang:1.20
+
+WORKDIR /go/src/sigs.k8s.io/scheduler-plugins
+COPY . .
+ARG ARCH
+ARG RELEASE_VERSION
+RUN RELEASE_VERSION=${RELEASE_VERSION} make build-scheduler.$ARCH
+
+FROM $ARCH/alpine:3.16
+
+ADD ./manifests/scheduler-config.yaml /home/scheduler-config.yaml
+COPY --from=0 /go/src/sigs.k8s.io/scheduler-plugins/bin/kube-scheduler /bin/kube-scheduler
+
+WORKDIR /bin
+CMD ["kube-scheduler"]
```

Comment:
Can you elaborate how this Dockerfile is invoked? I don't see any changes in the Makefile.

Reply:
Hi Parul, you can build from this https://github.com/sustainable-computing-io/scheduler-plugins/pull/1

Reply:
Why do we have >5000 files in this pull request?